src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square() {
+    }
+
+    public Square(double side) {
+        this.typeOfFigure = TypeOfFigure.SQUARE;
```

Comment:
Don't use static methods in your solution. It seems like getQuadrangularArea might be static. Instead, you should create an instance method that calculates the area.